spec/genre_spec.rb

```diff
@@ -0,0 +1,64 @@
+require './classes/genre'
```

Comment:
### **Hi @brytebee,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 

### **Optional**
- Please always send a PR that contains details about your approach to the project, and it will be a plus if you can make it very intuitive about the project.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._